front/lib/connector_providers.ts

```diff
@@ -224,6 +240,9 @@ export const isConnectorProviderAllowedForPlan = (
     case "snowflake":
       // TODO(SNOWFLAKE): Add a isSnowflakeAllowed column to the plan model.
       return true;
+    case "zendesk":
+      // TODO(ZENDESK): Add a isZendeskAllowed column to the plan model.
```

Comment:
Let's not add another column please 😅.

Reply:
yeah agreed, i'll remove the todo

Reply:
So we're dropping the idea of enabling connections per plan type?

Reply:
I guess it's fine to lazily add them as needed. But we're indeed not using them much for now.